signingscript/task.py

```diff
@@ -190,24 +209,37 @@ def get_suitable_signing_servers(signing_servers, cert_type, signing_formats):
             zipalign_command += ['-v']
 
         zipalign_command += [_ZIP_ALIGNMENT, original_apk_location, temp_apk_location]
-        await _execute_subprocess(zipalign_command)
+        await utils._execute_subprocess(zipalign_command)
         shutil.move(temp_apk_location, abs_to)
 
     log.info('"{}" has been zip aligned'.format(abs_to))
 
 
-async def _execute_subprocess(command):
-    log.info('Running "{}"'.format(' '.join(command)))
-    subprocess = await asyncio.create_subprocess_exec(*command, stdout=PIPE, stderr=STDOUT)
-    log.info("COMMAND OUTPUT: ")
-    await utils.log_output(subprocess.stdout)
-    exitcode = await subprocess.wait()
-    log.info("exitcode {}".format(exitcode))
+# _explode_dmg {{{1
+async def _explode_dmg(context, from_):
```

Comment:
Bikeshed: How about `_convert_dmg_to_tar_gz()`? 

It may be my own misunderstanding of the English language. To me, "explode" means a raw material has been shattered without any additional post-processing. Hence, I find weird the function tars up the results, after the actual extraction.